CMakeLists.txt

```
@@ -77,6 +77,7 @@ option(MUE_BUILD_CONVERTER_MODULE "Build converter module" ON)
 option(MUE_BUILD_DIAGNOSTICS_MODULE "Build diagnostic code" ON)
 option(MUE_BUILD_IMPORTEXPORT_MODULE "Build importexport module" ON)
 option(MUE_BUILD_VIDEOEXPORT_MODULE "Build videoexport module" OFF)
+option(MUE_BUILD_IMAGEEXPORT_MODULE "Build videoexport module" ON)
```

```suggestion
option(MUE_BUILD_IMAGEEXPORT_MODULE "Build imageexport module" ON)
```
Or, to keep the name of the option consistent with the name of the module:
```suggestion
option(MUE_BUILD_IMAGESEXPORT_MODULE "Build imagesexport module" ON)
```

---

Thank you, I noticed too, but late ..
Let me fix it in another PR, so as not to wait for building ...